tests/test_all_objects.py

```diff
@@ -0,0 +1,28 @@
+# Navigate to working directory in command line
+# Run:   test_all_objects.py
+# Great video for help: https://www.bing.com/videos/riverview/relatedvideo?q=python+what+are+unit+tests&mid=559EF026DB1A3B82BD0E559EF026DB1A3B82BD0E&FORM=VIRE
+# Useful unit testing docs: https://docs.python.org/3/library/unittest.html#unittest.TestCase.debug
+
+import unittest
+import all_objects
+
+
+class TestDevModule(unittest.TestCase):
```

Comment:
Like we talked about, this is a tricky test, because it depends on your internet connection

Reply:
The keyword here is "mocking," e.g. https://changhsinlee.com/pytest-mock/